README.md

```diff
@@ -39,6 +39,14 @@ input.addEventListener("change", function(e) {
         },
         onSuccess: function() {
             console.log("Download %s from %s", upload.file.name, upload.url)
+        },
+        onShouldRetry: function(err, retryAttempt, options) {
```

Comment:
Please move this example to its own section in docs/usage.md. The example in the README.md is meant as the  minimum useful code, so we try to avoid adding too many options here.